connector/kafka-0-10-sql/src/test/resources/error/kafka-error-classes.json

```diff
@@ -0,0 +1,26 @@
+{
+  "TOPIC_PARTITIONS_IN_END_OFFSET_ARE_NOT_SAME_WITH_PREFETCHED" : {
```

Comment:
Since this could appear in headers of paragraphs in docs, could you make it shorter if it is possible. Also cc @srielau

Reply:
I'm not sure about the doc since this is not a part of Spark's centralized error class. I intended to separate this one for Kafka data source. Kafka data source is considered as a reference implementation hence I wanted to see how 3rd party can integrate Spark's error class framework.

Reply:
Still a mouthful..

Reply:
I just went with MISMATCHED_TOPIC_PARTITIONS_BETWEEN_END_OFFSET_AND_PREFETCHED although it is still lengthy. Probably I can't change this till I understand what is the purpose and what's preference, via getting answers from above two questions.